<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>120</width>
      <height>261</height>
     </rect>
    </property>
    <widget class="QPushButton" name="pass_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(218, 218, 218);</string>
     </property>
     <property name="text">
      <string>Наши пассажиры</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../.designer/backup/Sign-in-icon.jpg</normaloff>../../../.designer/backup/Sign-in-icon.jpg</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="air_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(218, 218, 218);</string>
     </property>
     <property name="text">
      <string>Наши рейсы</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../.designer/backup/icons/мои поездки.png</normaloff>../../../.designer/backup/icons/мои поездки.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="rasp_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(218, 218, 218);</string>
     </property>
     <property name="text">
      <string>Расписание поездов</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../.designer/backup/icons/история.png</normaloff>../../../.designer/backup/icons/история.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="data_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(218, 218, 218);</string>
     </property>
     <property name="text">
      <string>Мои личные данные</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../.designer/backup/icons/img_148969.png</normaloff>../../../.designer/backup/icons/img_148969.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="upload_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(218, 218, 218);</string>
     </property>
     <property name="text">
      <string>Выгрузить в EXEL</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../.designer/backup/icons/история.png</normaloff>../../../.designer/backup/icons/история.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>481</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../.designer/backup/1613679178_4-p-fon-dlya-prezentatsii-rzhd-4.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>-52</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>админ.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="hello_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Добрый день, </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>-10</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>logo-rgd.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>311</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
